Grbl_Esp32/grbl_trinamic.cpp

```diff
@@ -252,21 +396,56 @@ void trinamic_stepper_enable(bool enable) {
 #ifdef X_TRINAMIC
     TRINAMIC_X.toff(toff);
 #endif
+#ifdef X2_TRINAMIC
+    TRINAMIC_X2.toff(toff);
+#endif
+
 #ifdef Y_TRINAMIC
     TRINAMIC_Y.toff(toff);
 #endif
+#ifdef Y2_TRINAMIC
+    TRINAMIC_Y2.toff(toff);
+#endif
+
 #ifdef Z_TRINAMIC
     TRINAMIC_Z.toff(toff);
 #endif
+#ifdef Z2_TRINAMIC
+    TRINAMIC_Z2.toff(toff);
+#endif
+
 #ifdef A_TRINAMIC
     TRINAMIC_A.toff(toff);
 #endif
+#ifdef A2_TRINAMIC
+    TRINAMIC_A2.toff(toff);
+#endif
+
 #ifdef B_TRINAMIC
     TRINAMIC_B.toff(toff);
 #endif
+#ifdef B2_TRINAMIC
+    TRINAMIC_B2.toff(toff);
+#endif
+
 #ifdef C_TRINAMIC
     TRINAMIC_C.toff(toff);
 #endif
+#ifdef C2_TRINAMIC
+    TRINAMIC_C2.toff(toff);
+#endif
+
+}
+
+// returns the next spi index. We cannot preassign to axes because ganged (X2 type axes) might
+// need to be inserted into the order of axes.
+uint8_t get_next_trinamic_driver_index() {
+    static uint8_t index = 1; // they start at 1
+    #ifndef TRINAMIC_DAISY_CHAIN
```

Comment:
For me, it is easier to understand ifdef/else/then than ifndef/else/then because I don't have to hold the inversion in my head while thinking about the code.  I typically reserve ifndef for else-less constructs.  In that case I think "this goes away" and the inversion problem doesn't bother me.